test/paypro.js

```diff
@@ -4,12 +4,12 @@ var _ = require('lodash');
 var chai = chai || require('chai');
 var sinon = sinon || require('sinon');
 var should = chai.should();
-var PayProReq = require('../lib/payprorequest');
+var PayPro = require('../lib/paypro');
 var TestData = require('./testdata');
 
 
-describe('payprorequest', function() {
-  var xhr, http;
+describe('paypro', function() {
```

Comment:
I think we need to have a test in place to check that the address we pass as refund address is actually ours. This should be an extra check to the `broadcastTxProposal` function